Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,518 @@
+            url_suffix=url_suffix,
+            json_data=request_data
+        )
+        return data
+
+    def list_files(self, url_suffix, request_data):
+        data = self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            json_data=request_data
+        )
+        return data
+
+    def list_incidents(self, filters, limit):
+        return self._http_request(
+            method='POST',
+            url_suffix='/alerts/',
+            json_data={
+                'filters': filters,
+                'limit': limit
+            }
+        )
+
+
+def arg_to_timestamp(arg):
+    if isinstance(arg, str) and arg.isdigit():
+        return int(arg)
+    if isinstance(arg, str):
+        date = dateparser.parse(arg, settings={'TIMEZONE': 'UTC'})
+        return int(date.timestamp())
+    if isinstance(arg, (int, float)):
+        return int(arg)
+
+
+def demisto_severity_to_api_severity(severity):
+    severity_options = {
+        'Low': 0,  # low severity
+        'Medium': 1,  # medium severity
+        'High': 2  # high severity
+    }
+    return severity_options[severity]
+
+
+def demisto_resolution_status_to_api_resolution_status(resolution_status):
+    resolution_status_options = {
+        'Open': 0,
+        'Dismissed': 1,
+        'Resolved': 2
+    }
+    return resolution_status_options[resolution_status]
+
+
+def demisto_source_type_to_api_source_type(source):
+    source_type_option = {
+        'Access_control': 0,
+        'Session_control': 1,
+        'App_connector': 2,
+        'App_connector_analysis': 3,
+        'Discovery': 5,
+        'MDATP': 6
+    }
+    return source_type_option[source]
+
+
+def demisto_file_type_to_api_file_type(file_type):
+    file_type_option = {
+        'Other': 0,
+        'Document': 1,
+        'Spreadsheet': 2,
+        'Presentation': 3,
+        'Text': 4,
+        'Image': 5,
+        'Folder': 6
+    }
+    return file_type_option[file_type]
+
+
+def demisto_sharing_options_to_api_sharing_options(sharing):
+    file_sharing_options = {
+        'Private': 0,
+        'Internal': 1,
+        'External': 2,
+        'Public': 3,
+        'Public_Internet': 4
+    }
+    return file_sharing_options[sharing]
+
+
+def demisto_ip_category_to_api_ip_category(ip_category):
+    ip_category_option = {
+        'Corporate': 1,
+        'Administrative': 2,
+        'Risky': 3,
+        'VPN': 4,
+        'Cloud_provider': 5,
+        'Other': 6
+    }
+    return ip_category_option[ip_category]
+
+
+def demisto_is_external_to_api_is_external(is_external):
+    is_external_option = {
+        'External': True,
+        'Internal': False,
+        'No_value': None
+    }
+    return is_external_option[is_external]
+
+
+def demisto_status_options_to_api_status_options(status):
+    status_options = {
+        'N/A': 0,
+        'Staged': 1,
+        'Active': 2,
+        'Suspended': 3,
+        'Deleted': 4,
+    }
+    return status_options[status]
+
```

Comment:
You have a lot of functions which are basically constants, why not replace them with real constants?
I.e.
STATUS_OPTIONS = {
        'N/A': 0,
        'Staged': 1,
        'Active': 2,
        'Suspended': 3,
        'Deleted': 4,
    }

then where you use it STATUS_OPTIONS[status]

and better give them a more meaningful name